src/actions/cmsActions.js

```diff
@@ -40,12 +41,17 @@ export const getTutorialDataAction = () => async (dispatch: Dispatch, getState:
   try {
     const response: CmsData = await prismicClient.query(Predicates.any(DOCUMENT_TYPE, [NATIVES, NEWBIES]));
     const tutorialData = getTutorialDataObject(response);
-    if (!isValidTutorialData(tutorialData)) return;
+    if (!isValidTutorialData(tutorialData)) {
+      log.info('user got invalid tutorial data');
```

Comment:
@mdjastrzebski I've just revisited this task which was originally created 7 months ago, and re-written it. In short - use `reportLog` and `reportErrorLog`. FYI the logger in `utils` is captured in Sentry, but it looks like the implementation of the `utils/logger.js` was unfinished.

Reply:
@IAmKio if we use `reportLog` and `reportErrorLog` this will send Sentry event/error on each invocation. From context it seems that we would rather want to created breadcrumbs in most cases, and send events/errors only for terminal cases. WDYT?

Reply:
@mdjastrzebski just having a look through the documentation at the available event types to send along with each Sentry event. It appears that to log effectively with a log trail, we'll need to follow this documentation which will require a small extension of the `reportLog` function in `common.js`: https://docs.sentry.io/platforms/react-native/enriching-events/breadcrumbs/

Reply:
I'll update the ticket to reflect this

Reply:
IMO I would suggest keeping the `reportLog` as is, meaning that it records event to Sentry, while adding `logBreadcrumb` or whatever for the breadcrumbs. This should make code structure more natural.

Reply:
@mdjastrzebski i believe this is the solution we settled on. I wasn't keen on stuffing `reportLog` with more parameters also.